ci-operator/step-registry/aws/provision/bastionhost/proxy-whitelist/aws-provision-bastionhost-proxy-whitelist-commands.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+REGION=${LEASED_RESOURCE}
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+
+function run_command() {
+    local cmd="$1"
+    echo "Running Command: ${cmd}"
+    eval "${cmd}"
+}
+
+function run_ssh_cmd() {
+    local sshkey=$1
+    local user=$2
+    local host=$3
+    local remote_cmd=$4
+    local options=$5
+    local cmd
+    options+=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes "
+    cmd="ssh ${options} -i \"${sshkey}\" ${user}@${host} \"${remote_cmd}\""
+    run_command "$cmd" || return 2
+    return 0
+}
+
+function run_scp_to_remote() {
+    local sshkey=$1
+    local user=$2
+    local host=$3
+    local src=$4
+    local dest=$5
+    local options=$6
+    local cmd
+    options+=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o VerifyHostKeyDNS=yes "
+    cmd="scp ${options} -i \"${sshkey}\" ${src} ${user}@${host}:${dest}"
+    run_command "$cmd" || return 2
+    return 0
+}
+
+proxy_host_address=$(head -n 1 "${SHARED_DIR}/bastion_public_address")
+proxy_host_user=$(head -n 1 "${SHARED_DIR}/bastion_ssh_user")
+
+
+# https://docs.openshift.com/container-platform/4.15/installing/install_config/configuring-firewall.html#configuring-firewall
+
+# Registry URLs
+cat <<EOF > ${ARTIFACT_DIR}/allowlist.txt
+access.redhat.com
+cdn.quay.io
+cdn01.quay.io
+cdn02.quay.io
+cdn03.quay.io
+quay.io
+registry.redhat.io
+sso.redhat.com
+EOF
+
+# Telemetry
+cat <<EOF >> ${ARTIFACT_DIR}/allowlist.txt
+api.access.redhat.com
+cert-api.access.redhat.com
+console.redhat.com
+infogw.api.openshift.com
+EOF
+
+# AWS
+cat <<EOF >> ${ARTIFACT_DIR}/allowlist.txt
+*.cloudfront.net
+aws.amazon.com
+ec2.${REGION}.amazonaws.com
+ec2.amazonaws.com
+elasticloadbalancing.${REGION}.amazonaws.com
+events.amazonaws.com
+iam.amazonaws.com
+route53.amazonaws.com
+s3.${REGION}.amazonaws.com
+s3.amazonaws.com
+s3.dualstack.${REGION}.amazonaws.com
+servicequotas.${REGION}.amazonaws.com
+sts.${REGION}.amazonaws.com
+sts.amazonaws.com
+tagging.${REGION}.amazonaws.com
+tagging.us-east-1.amazonaws.com
+EOF
+
+# Operator
+# *.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
+cat <<EOF >> ${ARTIFACT_DIR}/allowlist.txt
+oauth-openshift.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
+console-openshift-console.apps.${CLUSTER_NAME}.${BASE_DOMAIN}
+api.openshift.com
+console.redhat.com
+mirror.openshift.com
+quayio-production-s3.s3.amazonaws.com
+rhcos.mirror.openshift.com
+sso.redhat.com
```

Comment:
This one is duplicated with the line 60

Reply:
I followed sections in https://docs.openshift.com/container-platform/4.15/installing/install_config/configuring-firewall.html#configuring-firewall, `sso.redhat.com` is required by both registry and operator,  I think it's easy to maintain and check against official doc